backend/learnings/views.py

```diff
@@ -246,14 +246,9 @@ class GetUserActivityLog(APIView):
     permission_classes = [IsAuthenticated]
     authentication_classes = [TokenAuthentication]
 
-    def get(self, request, page, username):
+    def get(self, request, username):
         user = EUser.objects.get(username=username)
-        if page == "dashboard":
-            activities = UserActivityLog.objects.filter(user_id=user).order_by("-created_at")
-        else:
-            activities = UserActivityLog.objects.filter(
-                activity_description="quiz", user_id=user
-            ).order_by("-created_at")
+        activities = UserActivityLog.objects.filter(user_id=user).order_by("-created_at")
```

Comment:
why is the checking if the user is in dashboard or profile page removed?

Reply:
I did not notice this back then but this is the intended behavior, is this working correctly?
```
In profile page:
  display the activities of a specific user
In dashboard
  display the activities of a specific user and the users they follow
```

Reply:
Hindi na ngayon sir kasi tinaggal na sya so bali kung ano nasa dashboard yun din yung nasa profile

Reply:
please put it back po :bow: 
```
In profile page:
  display the activities (quiz taken, follow, unfollow) of a specific user
In dashboard
  display the activities (quiz taken, follow, unfollow) of a specific user and the users they follow
```

Reply:
its ok to do this tomorrow since it is already 7pm :bow:

Reply:
done sir, na push force ko po nakalimutan ko mag create pala dapat ng new commit